packages/components/src/components/input-field/input-field.scss

```diff
@@ -60,7 +60,7 @@ button.dc-input-wrapper {
             cursor: pointer;
 
             & ~ .dc-input-wrapper__input {
-                @extend .input:hover;
+                @extend .input-hover;
```

Comment:
Extending Compound classes ([doc](https://sass-lang.com/documentation/breaking-changes/extend-compound)) is deprecated and not supported in `sass`.